index.js

```diff
@@ -0,0 +1,16 @@
+let age = 30;
+let firstname = "Nikhil";
+let coursecomplete = 'True';
```

Comment:
if the value is a boolean, then you can set the value without quotes. Also, consider using camel casing when naming variables

`let courseComplete = true;`
`let courseComplete = false;`